Iterate on community page

bhellema opened the pull request:
Fix #43

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/new-homes/idaho/boise-metro/nampa/adams-ridge
- After: https://issue-43--hubblehomes-com--aemsites.hlx.page/new-homes/idaho/boise-metro/nampa/adams-ridge

codevoyage commented:
The template is missing these buttons:
<img width="643" alt="Screenshot 2024-06-10 at 1 16 27 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/7455b910-64af-4df7-9c10-7637ae5554fd">



The styling of the community name is different + the state is missing:
<img width="380" alt="Screenshot 2024-06-10 at 1 17 56 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/529f44f2-d059-4935-9014-aadc742dd8b8">


The links in the description are styled differently, they are not blue but black. 
<img width="281" alt="Screenshot 2024-06-10 at 1 20 55 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/5a3b90f4-0754-490f-926a-b2942f9ef

Maybe this will come later, but the sidebar is supposed to have  Home Specialist information and Sales Center Location:
<img width="306" alt="Screenshot 2024-06-10 at 1 21 54 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/d6117d50-f56e-476f-9160-434c62a79c45">
97d">

The share and save buttons are missing text at the bottom: 
<img width="220" alt="Screenshot 2024-06-10 at 1 23 00 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/575d4eb8-18a8-4a6c-b6a0-7d7fcf3a31c8">

The text underneath the sales specialist should be centered. 
<img width="554" alt="Screenshot 2024-06-10 at 1 23 56 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/c34a2ad2-f5c8-4f15-9fff-12b8b1bc2bc4">

The names of the models above the cards are capitalized:
<img width="1207" alt="Screenshot 2024-06-10 at 1 24 54 PM" src="https://github.com/aemsites/hubblehomes-com/assets/6240431/7ce9c5e5-4c72-4699-8827-58375e1a4463">

codevoyage commented on templates/community/community.css:
```diff
@@ -0,0 +1,141 @@
+    display: none;
+  }
+
+  .action-bar {
```
The action-bar CSS can be moved to layouts.css since other templates also have it.

bhellema replied:
This should be moved into it's own template block later on.

codevoyage commented on blocks/models/cards/card.css:
```diff
@@ -45,10 +45,10 @@
+        /*.btn-primary2 {*/
+        /*  padding: 3px 12px;*/
+        /*  display: block;*/
+        /*}*/
```
Commented code can be removed.

codevoyage commented on styles/layouts.css:
```diff
@@ -0,0 +1,241 @@
+    }
+
+    /* bring the aside below the content */
+    @media (width <= 1280px) {
```
In the original website, the aside moves down at 991 px.

bhellema replied:
Line 145 takes care of that.  I'll remove the comment to remove confusion here.

codevoyage commented:
A few UI differences from the original:
1. The share icon is different.
2. The position of the share and the save button is different. 
3. The phone icon used in the specialist block is facing the other way. 
4. The link in the disclaimer is the wrong colour. 
5. Placement of the mail icon in the specialist block.
6. The content of the sidebar containing sales agent's information is different.

codevoyage commented on templates/community/community.js:
```diff
@@ -0,0 +1,304 @@
+    content.push(['photo', specialist.headshotImage]);
+    content.push(['email', specialist.email]);
+
+    const specialistsBlock = buildBlock('specialists', content);
```
Wouldn't it be better to have the buildBlock and decorateBlock functions in templates/blocks/specialists.js and return the block element already decorated? This way, the block creation code is tucked away in its file, and the caller of the function appends the block to the template.

bhellema commented:
> A few UI differences from the original:
> 1. The share icon is different.  
- yes unless we pull in font awesome this is the free svg that we get

> 2. The position of the share and the save button is different.
- fixed
> 3. The phone icon used in the specialist block is facing the other way.
- fixed, needed to change the way the svg was added so that I can scaleX it.
> 4. The link in the disclaimer is the wrong colour.
- global discussion to have about authored links
> 5. Placement of the mail icon in the specialist block.
- mine is nicer :D
> 6. The content of the sidebar containing sales agent's information is different.
- not focusing on this in this pr.
> 7. The button colours and button text in the cards are different.
- fixed with updated buttons

codevoyage reviewed (approved):
There are a couple minor comments left in the code that you can check out if you want.
